src/main/java/com/nooul/apihelpers/springbootrest/controllerAdvices/GlobalExceptionAdvice.java

```diff
@@ -1,158 +0,0 @@
-package com.nooul.apihelpers.springbootrest.controllerAdvices;
```

Comment:
I need to include that too - although I will probably drop the NotFoundException since I liked the way you throw ResponseStatus Exceptions. I have so many projects that have this as a dependency so it is not an option

Reply:
Got it.